pydata_sphinx_theme/theme.conf

```diff
@@ -21,3 +21,8 @@ search_bar_position = sidebar
 navigation_with_keys = True
 show_toc_level = 1
 navbar_align = content
+use_version_switch = True
```

Comment:
I think this (and enable_locale) should probably default to False start as an opt-in in case other people are already rolling their own version switcher?